Dockerfile

```diff
@@ -61,8 +57,8 @@ RUN apk add --upgrade --no-cache apk-tools busybox && \
     apk update && \
     apk add --no-cache gcc g++ python3-dev py3-pip musl-dev libffi-dev openssl-dev && \
     apk -U upgrade --no-cache && \
-    pip3 install --no-cache-dir -U pip && \
-    pip3 install --no-cache-dir -r requirements.txt
+    pip3 install --no-cache-dir -U pip
+RUN pip3 install --no-cache-dir -r requirements.txt
```

Comment:
Why did you make this a separate layer?

Reply:
For iterative development, each stage is cached individually. Thus, allowing the requirements install be on its own line means developers save a ton of time when they decide to iterate.